r124/outref/ydb312_gtm8182c.txt

```diff
@@ -0,0 +1,62 @@
+Test A: INST3-INST4 replicating instance is on
```

Comment:
replicating instance is on -> jnlpool is up on INST3
Similar change to the off case in line 34